### DISCUSSION TITLE
Editable Dropdown in Table

### DISCUSSION THREAD

**Discussion by Jorg71** [quasarframework/quasar#11649] [General - Components / Directives / etc]
Hi there,

in the Table examples I can't find any dropdown.

Is it possible to use an editable dropdown in a table?

For example:
- Collumn Name: Region
- Dropdown with values "EU", "US", "CN"

The values in the dropdown must be bound to a database.

Additional to that: If there would an option for filtering the table by a value from the dropdown (also by dropdown), this would be great.

Could you please help me with that?
Jorg

**Comment by metalsadman** [comment]
use q-select on one of the cell slot.
